### ISSUE TITLE
Does it support image data preview and label modification?

### ISSUE THREAD

**Issue by becauseofAI** [mage-ai/mage-ai#436]

**Comment by tommydangerous** [comment]
Hey @becauseofAI  , so sorry for the super long delay. We’ve been hard at work improving the tool for you.

We’ve actually changed it quite drastically to now support writing your own custom code in the tool.

With Mage, you can now transform your data and build production ready pipelines for ML.

Please take a look and go through the setup. If you have any issues, please ping me here, email me at [tommy@mage.ai](mailto:tommy@mage.ai), or please join our Slack and chat with me there: https://www.mage.ai/chat.

TLDR: yes, it supports image data preview and label modification now.
